docs/user/guides/access/wifi.md

```diff
@@ -1,27 +1,25 @@
 ---
-title: WI-FI
+title: Wi-Fi
 sidebar_position: 1
 ---
 
-# Wifi Hotspot
+If you have bought a DAppNode from our store or you installed DAppNode in a physical device with a Wi-Fi chip inside, you can directly connect to your DAppNodes UI by connecting your client device to the DAppNode's Wi-Fi hotspot.
```

Comment:
need to note this does not just work with every wifi card or even every NUC, it can be made to work if it doesn't, in command line.  But it often doesn't work on hardware not from us. 
```suggestion
If you have bought a DAppNode from our store or you installed DAppNode in a physical device with a Wi-Fi chip inside, you can directly connect to your DAppNodes UI by connecting your client device to the DAppNode's Wi-Fi hotspot.
_If you did not purchase your hardware from us there is a possibility the built in Wi-Fi card may not be recognized by the OS on install and will need to be activated manually, this varies greatly depending on what hardware is used if you encounter an issue please reach out to support on [Discord!](https://discord.gg/dappnode/)_
```

Reply:
Ah wow, I had this in my drafts but forgot about it. Good catch!